lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -27,12 +27,15 @@ class WagnGenerator < Rails::Generators::AppBase
   public_task :create_root
   
 ## should probably eventually use rails-like AppBuilder approach, but this is a first step.  
-  def dev_setup
-    @wagn_path = options['gem-path']
+  def dev_setup  
     if options['core-dev']
-      @wagn_path ||= ask("Enter the path to your local wagn gem installation: ")
+      @wagn_path = options['gem-path'] || ask("Enter the path to your local wagn gem installation: ")
```

Comment:
I liked this outside of the core-dev block; it's a nice shortcut for using a locally modified gem even when not doing core dev per se.